diff --git a/lib/nodes/index.cjs b/lib/nodes/index.cjs
index 43d2d6c53d453235b105827dc166798b30d13f4e..236ae6cb5b1f40a64b16fb0ab81f2a0ae6603921 100644
--- a/lib/nodes/index.cjs
+++ b/lib/nodes/index.cjs
@@ -121,12 +121,7 @@ var rimMultiply = (0, import_tsl4.nodeImmutable)(THREE3.PropertyNode, "vec3").to
 var matcap = (0, import_tsl4.nodeImmutable)(THREE3.PropertyNode, "vec3").toVar("matcap");
 var parametricRim = (0, import_tsl4.nodeImmutable)(THREE3.PropertyNode, "vec3").toVar("ParametricRim");
 var FnCompat = (jsFunc) => {
-  const threeRevision2 = parseInt(THREE_WEBGPU.REVISION, 10);
-  if (threeRevision2 >= 168) {
-    return THREE_TSL.Fn(jsFunc);
-  } else {
-    return THREE_WEBGPU.tslFn(jsFunc);
-  }
+  return THREE_TSL.Fn(jsFunc);
 };
 var linearstep = FnCompat(
   ({
diff --git a/lib/nodes/index.module.js b/lib/nodes/index.module.js
index 34a1a490707e0dd4ae4334261ccb813c113cd539..6563bc94a1cb5e9493b3594d2a02a0cbd460c5a0 100644
--- a/lib/nodes/index.module.js
+++ b/lib/nodes/index.module.js
@@ -99,12 +99,7 @@ var rimMultiply = nodeImmutable(THREE3.PropertyNode, "vec3").toVar("RimMultiply"
 var matcap = nodeImmutable(THREE3.PropertyNode, "vec3").toVar("matcap");
 var parametricRim = nodeImmutable(THREE3.PropertyNode, "vec3").toVar("ParametricRim");
 var FnCompat = (jsFunc) => {
-  const threeRevision2 = parseInt(THREE_WEBGPU.REVISION, 10);
-  if (threeRevision2 >= 168) {
-    return THREE_TSL.Fn(jsFunc);
-  } else {
-    return THREE_WEBGPU.tslFn(jsFunc);
-  }
+  return THREE_TSL.Fn(jsFunc);
 };
 var linearstep = FnCompat(
   ({
